docker-compose.yml

```diff
@@ -53,41 +52,40 @@ services:
       - ./data/certbot/conf/:/etc/letsencrypt
       - ./data/certbot/www/:/var/www/certbot
     entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
+    depends_on:
+      - nginx
 
   greenlight:
     entrypoint: [bin/start]
-    image: ${GREENLIGHT_DOCKER_IMAGE:-bigbluebutton/greenlight:v3}
+    image: $GREENLIGHT_DOCKER_IMAGE
     container_name: greenlight
     restart: unless-stopped
     logging:
       driver: journald
     env_file: ./data/greenlight/.env
     volumes:
       - ./data/greenlight/bin/start:/usr/src/app/bin/start
+      - ./data/greenlight/storage:/usr/src/app/storage
     depends_on:
       - postgres
       - redis
 
   keycloak:
-    image: jboss/keycloak:16.1.1
+    image: $KEYCLOAK_DOCKER_IMAGE
```

Comment:
We want to default to something here, because the ENV variables work different across versions. The variables we are using are for the legacy images.

Change it to
`image: ${KEYCLOAK_DOCKER_IMAGE:-quay.io/keycloak/keycloak:19.0.3-legacy}`

Also add a note to the dotenv file, so this provision is documented

Reply:
Wasn't aware of the brake changes in configs,
Thanks for the heads up!